xc/xc7/tests/iosettings/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
-#add_custom_target(all_xc7_iosettings_diff_fasm)
+# These targets make Kokoro complain that the Ninja version is too low.
+#add_custom_target(all_xc7_iosettings_generate)
+#add_custom_target(all_xc7_iosettings_bin)
+#add_custom_target(all_xc7_iosettings_bit_v)
+#add_custom_target(all_xc7_iosettings_vivado_diff_fasm)
 
-function(xc7_iosettings_design MODE BOARD IOSTANDARD DRIVES SLEWS TERMS)
+function(xc7_iosettings_design MODE BOARD IOSTANDARD DRIVES SLEWS TERMS VREF)
```

Comment:
Please convert this function to use named arguments calling convention, using cmake_parse_arguments.  It is hard to parse a function with 7 arguments.

Reply:
Ok, I've cleaned that up.